style.css

```diff
@@ -1,5 +1,6 @@
 body {
   margin: 0px;
+  margin: 0vw;
```

Comment:
Didn't we agree to remove units?

Reply:
Did we?

Reply:
![](https://m1guelpf.joined-the.club/oZJzfgbnu.png)

Reply:
I'll get right on it!

Reply:
I removed them but kept the other lines as fallbacks.

Reply:
Why would you need fallback?